scss/utilities/_position.scss

```diff
@@ -23,3 +23,8 @@
     z-index: $zindex-sticky;
   }
 }
+
+.pos-s { position: static; }
```

Comment:
At that point, we might as well use the full value names. `.p-sticky`, `.p-static`, etc.